src/components/LibraryContainer.tsx

```diff
@@ -72,12 +75,13 @@ export class LibraryContainer extends React.Component<LibraryContainerProps, Lib
         this.handleKeyDown = this.handleKeyDown.bind(this);
         this.scrollToExpandedItem = this.scrollToExpandedItem.bind(this);
         this.setTooltipText = this.setTooltipText.bind(this);
+        this.setHostContext = this.setHostContext.bind(this);
 
         // Set handlers after methods are bound.
         this.props.libraryController.setLoadedTypesJsonHandler = this.setLoadedTypesJson;
         this.props.libraryController.setLayoutSpecsJsonHandler = this.setLayoutSpecsJson;
         this.props.libraryController.refreshLibraryViewHandler = this.refreshLibraryView;
-
+        this.props.libraryController.setHostContextHandler = this.setHostContext;
```

Comment:
I don't see these methods (setHostContext) called anywhere ? are they called on the c#/webview2 side ?

Reply:
I see it now.

Reply:
yes, see the dynamo pr.
https://github.com/DynamoDS/Dynamo/pull/15204